pkg/deploy/registry/init_test.go

```diff
@@ -9,7 +9,7 @@
 // Contributors:
 //   Red Hat, Inc. - initial API and implementation
 //
-package plugin_registry
+package registry
```

Comment:
The same as above: we should distinguish between plugin and devfile registries, imo.

Reply:
Moved to different packages.